app/models/user.rb

```diff
@@ -51,8 +51,6 @@ def as_json(options = {})
     super(only: JSON_ATTRIBUTES)
   end
 
-  private
-
   def self.find_first_by_auth_conditions(warden_conditions)
```

Comment:
I wonder if `find_first_by_auth_conditions` is still being used. I could not find any references to it... perhaps it's used by something that's outside the repository?